### PR TITLE
Trim account name if it contains a '.'

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
SNOW-257844

### CODE REVIEW

File: src/main/java/net/snowflake/ingest/connection/SecurityManager.java [hunk 1]
```diff
@@ -120,6 +120,33 @@
...
+   * @return initial part of account name if originally it contained a ".", otherwise return the same accountName.
+   * @see <a href="https://github.com/snowflakedb/snowflake-connector-python/blob/master/src/snowflake/connector/util_text.py#L227">Python Connector</a>
+   */
+  private String parseAccount(final String accountName)
+  {
+    String parseAccount = null;
+    if (accountName.contains("."))
+    {
+      final String[] accountParts = accountName.split("[.]");
+      if (accountParts.length > 1)
```
Comment (sfc-gh-rramachandran) [inline src/main/java/net/snowflake/ingest/connection/SecurityManager.java:138]: NIT: Is there a valid scenario where this is false? Can we remove it?

**Review by sfc-gh-rramachandran — APPROVED** [review]
LGTM!

File: src/main/java/net/snowflake/ingest/connection/SecurityManager.java (hunk 1 above)
Comment (sfc-gh-japatel) [reply src/main/java/net/snowflake/ingest/connection/SecurityManager.java:138]: Very rare. In cases like they pass an accountname which ends in a dot may be?

**Review by sfc-gh-hchaturvedi — APPROVED** [review]
LGTM. Can you please ensure it is tested against a prod env before we merge / release it?

**Comment by sfc-gh-japatel** [comment]
> LGTM. Can you please ensure it is tested against a prod env before we merge / release it?

I have tested it on local machine. but not on an actual account which has a "." in its name. Do you have recommendation or a test account which has dot in name?

**Comment by sfc-gh-japatel** [comment]
Tested this on `testaccount.reg.snow...com`

**Pull request merged**